nibe/data/vvm310_vvm500.json

```diff
@@ -2604,7 +2604,11 @@
     "info": "Indicates active relays on the PCA Base card. Please refer to the wiring diagram for relay description. Binary encoded. 1=on, 0=off. Bit0=K4,Bit1=K3,Bit2=K2,Bit3=K1",
     "size": "u8",
     "factor": 1,
-    "name": "eb100-ep14-pca-base-relays-43514"
+    "name": "eb100-ep14-pca-base-relays-43514",
+    "mappings": {
+      "0": "off",
+      "1": "on"
+    }
```

Comment:
I think this might be wrong. I think these are bitfields, but maybe the csv has invalid max values?

Reply:
Yes. It is wrong. Dropped all mappings that are binary encoded.